js/main.js

```diff
@@ -109,29 +113,37 @@ if (isMac) {
  * Sets chrome authentication flags in electron
  */
 function setChromeFlags() {
-    
+
     log.send(logLevels.INFO, 'checking if we need to set custom chrome flags!');
-    
+
     // Read the config parameters synchronously
     let config = readConfigFileSync();
-    
+
     // If we cannot find any config, just skip setting any flags
     if (config && config !== null && config.customFlags) {
-        
+
         log.send(logLevels.INFO, 'Chrome flags config found!');
-        
-        // If we cannot find the authServerWhitelist config, move on
+
         if (config.customFlags.authServerWhitelist && config.customFlags.authServerWhitelist !== "") {
             log.send(logLevels.INFO, 'Setting auth server whitelist flag');
             app.commandLine.appendSwitch('auth-server-whitelist', config.customFlags.authServerWhitelist);
         }
-        
-        // If we cannot find the authNegotiateDelegateWhitelist config, move on
+
         if (config.customFlags.authNegotiateDelegateWhitelist && config.customFlags.authNegotiateDelegateWhitelist !== "") {
             log.send(logLevels.INFO, 'Setting auth negotiate delegate whitelist flag');
             app.commandLine.appendSwitch('auth-negotiate-delegate-whitelist', config.customFlags.authNegotiateDelegateWhitelist);
         }
-        
+
+        // ELECTRON-261: Windows 10 Screensharing issues. We set chrome flags
+        // to disable gpu which fixes the black screen issue observed on
+        // multiple monitors
+        if (config.customFlags.disableGpu) {
```

Comment:
Will "customFlags" be available for all builds? If not, would be good to add a safety check here.